torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -280,7 +284,8 @@ def __getstate__(self):
 
     def __setstate__(self, state):
         r"""
-        Takes a provided ``state`` and retrieves ``PowerSGDState``.
+        Take a provided ``state`` and retrieves ``PowerSGDState``.
```

Comment:
Maybe something like:
```suggestion
        Take a provided ``state`` and set to this ``PowerSGDState`` instance.
```